snapcraft/services/confdbschemas.py

```diff
@@ -18,27 +18,27 @@
 
 from __future__ import annotations
 
+import collections
 import textwrap
 from typing import Any, override
 
-from craft_application.util import dump_yaml
+import yaml
 
 from snapcraft import models
 from snapcraft.services import Assertion
 
-_CONFDB_SCHEMA_TEMPLATE = textwrap.dedent(
+# Template parts to be assembled
+_TPL_BASE = textwrap.dedent(
     """\
     account-id: {account_id}
-    name: {name}
-    # The revision for this confdb-schema
-    # revision: {revision}
-    {views}
-    {body}
-    """
+    name: {name}"""
 )
+_TPL_REVISION_COMMENT = "# The revision for this confdb-schema\n# revision: {revision}"
+# Body template - {body_block_manual} will be the manually constructed 'body: |' block
+_TPL_BODY = "{body_block_manual}"
 
-
-_CONFDB_SCHEMA_VIEWS_TEMPLATE = textwrap.dedent(
+# Default view template for new schemas
+_CONFDB_SCHEMA_VIEWS_TEMPLATE_DEFAULT = textwrap.dedent(
```

Comment:
This is fine as-is and doesn't need to be changed to add a new `summary` field.  Why change it to make it more complex?